proto/platform/oidc/v1/oidc.platform.proto

```diff
@@ -19,6 +19,26 @@ service SecurityTokenService {
       }
     };
   }
+
+  rpc GetAccessToken(GetAccessTokenRequest) returns (TokenPair) {
+    option (google.api.http) = {
+      post: "/sts/access_token"
+      additional_bindings {
+        get: "/sts/access_token"
+      }
+    };
+  }
+}
+
+message GetAccessTokenRequest {
+  repeated string aud = 1;
+  string scope = 2;
+  string refresh_token = 3;
```

Comment:
Should this be in the `Authorization` header?

Reply:
The current `ExchangeRequest` does it through a field

Reply:
Fixed